### DISCUSSION TITLE
Will there be an English specialized version as well?

### DISCUSSION THREAD

**Discussion by godspirit00** [fishaudio/Bert-VITS2#319] [General]
As there is a Chinese specialized version and a Japanese specialized version, will there be an English specialized version as well?
Really looking forward to that.
Thanks!

**Comment by Stardust-minus** [comment]
No plan.

**Reply by Tybost** [reply]
Very sad to hear this ;(
I'm curious how much better a specialized version is over the previous. Guess the EN side will never know. :C
